tealer/detectors/rekeyto.py

```diff
@@ -8,11 +8,23 @@
     DetectorType,
 )
 from tealer.teal.basic_blocks import BasicBlock
-from tealer.teal.instructions.instructions import Gtxn, Return, Int
+from tealer.teal.instructions.instructions import Gtxn, Return, Int, Txn, Global, Addr, Eq, Neq
 from tealer.teal.instructions.transaction_field import RekeyTo
+from tealer.teal.global_field import ZeroAddress
 
 if TYPE_CHECKING:
     from tealer.utils.output import SupportedOutput
+    from tealer.teal.instructions.instructions import Instruction
+
+
+def _is_rekey_check(ins1: "Instruction", ins2: "Instruction") -> bool:
+    """check if ins1 is txn RekeyTo and ins2 is global ZeroAddress or addr ..."""
```

Comment:
Needs  better doc!   Something that looks like the following (but is correct :) )

```
    """Utility function to check check RekeyTo property that...

    Args:
        ins1 (Instrcution): first instruction somewhere
        ins2 (Instruction): second appearing instruction
          
    Returns:
        bool: True if :ins1: is txn RekeyTo and :ins2: is global ZeroAddress or addr 
        """
        ```

Reply:
Yes it's probably time to agree on a doc coding standard, and apply it everywhere. However, this would be outside of this PR